ptvsd/wrapper.py

```diff
@@ -129,6 +130,45 @@ def using_format(self, fmt):
 str_handlers = pydevd_extutil.EXTENSION_MANAGER_INSTANCE.type_to_instance.setdefault(pydevd_extapi.StrPresentationProvider, [])  # noqa
 str_handlers.insert(0, SafeReprPresentationProvider._instance)
 
+PTVSD_FILE = 2
```

Comment:
Is this for future extensibility? I would expect the below to be a set, rather, since all the values are the same, and only the keys differ.

Also, why 2? Does it correspond to something inside pydevd? If so, this probably needs a comment.